todo/src/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+import { ADD_TODO } from '../actions';
+
+const ToDoReducer =  (state = [], action) => {
+  switch (action.type) {
+    case ADD_TODO:
+      const newToDo = {
+        value: action.payload,
+        completed: false
+      }
```

Comment:
It's really unconventional to create this newToDo object in the reducer.  I would recommend creating it either in your actions or better yet create it in your handleSubmit event handler and then pass it off to your actions which can be sent to your reducer as the payload

Then your ADD_TODO case will look much cleaner:
```
case ADD_TODO:
  return [...state, action.payload]
```
What you have now works but imagine that later you decided to scale the project and there ends up being 15 different switch cases.